### ISSUE TITLE
bundle exec jekyll serve : Failing

### ISSUE THREAD

**Issue by gautamkrishnar** [github/personal-website#86]
**bundle exec jekyll serve** is giving me this error
```
Configuration file: /Users/gautam/WebstormProjects/gautamkrishnar.github.io/_config.yml
            Source: /Users/gautam/WebstormProjects/gautamkrishnar.github.io
       Destination: /Users/gautam/WebstormProjects/gautamkrishnar.github.io/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
   GitHub Metadata: No GitHub API authentication could be found. Some fields may be missing or have incorrect data.
  Conversion error: Jekyll::Converters::Scss encountered an error while converting 'assets/css/style.scss':
                    Invalid US-ASCII character "\xE2" on line 5
jekyll 3.7.4 | Error:  Invalid US-ASCII character "\xE2" on line 5
```

**Comment by brandonrosage** [comment]
Is this an error when running the command on a fork of this repo? If so, can you point me to the repo and branch so I can try to replicate the problem?

**Comment by gautamkrishnar** [comment]
@brandonrosage 

https://github.com/gautamkrishnar/gautamkrishnar.github.io

I am using master branch...  Haven't changed anything on the repo....

**Comment by thekaveman** [comment]
This may be related to an issue in the `gh-pages` gem github/pages-gem#613. 

TLDR; a workaround could be setting `theme: null` in your `_config.yml` file.

**Comment by gautamkrishnar** [comment]
Thanks @thekaveman  will try that out...

**Comment by brandonrosage** [comment]
@gautamkrishnar Did you get it sorted?

**Comment by gautamkrishnar** [comment]
Thanks @thechickeneater looks like it is fixed. Closing this one for now.

CC: @brandonrosage
